mobile/library/java/org/chromium/net/impl/CronvoyLogger.java

```diff
@@ -0,0 +1,54 @@
+package org.chromium.net.impl;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import io.envoyproxy.envoymobile.engine.types.EnvoyLogger;
+
+final class CronvoyLogger implements EnvoyLogger {
+  private FileWriter mWriter;
+
+  public CronvoyLogger() {}
+
+  public void stopLogging() {
+    try {
+      if (mWriter != null) {
+        mWriter.close();
+        mWriter = null;
+      }
+    } catch (IOException e) {
+      System.err.println("Encountered " + e.toString() + " when trying to stopLogging");
```

Comment:
I'm not sure I have a better answer, but I wonder if there is something else we should use. Probably not?

Reply:
Let's use a Android log like [this](https://github.com/envoyproxy/envoy/blob/main/mobile/library/java/org/chromium/net/impl/CronvoyUrlRequest.java#L596)